pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -256,23 +178,9 @@ func (c *Cluster) enablePrometheusModule(clusterName string) error {
 	return nil
 }
 
-// Ceph docs about the dashboard module: http://docs.ceph.com/docs/luminous/mgr/dashboard/
-func (c *Cluster) configureDashboardModule(clusterName string, enable bool) error {
-	if enable {
-		if err := client.MgrEnableModule(c.context, clusterName, dashboardModuleName, true); err != nil {
-			return fmt.Errorf("failed to enable mgr dashboard module. %+v", err)
-		}
-	} else {
-		if err := client.MgrDisableModule(c.context, clusterName, dashboardModuleName); err != nil {
-			return fmt.Errorf("failed to disable mgr dashboard module. %+v", err)
-		}
-	}
-	return nil
-}
-
 func getKeyringProperties(name string) (string, []string) {
 	username := fmt.Sprintf("mgr.%s", name)
-	access := []string{"mon", "allow *"}
+	access := []string{"mon", "allow profile mgr", "mds", "allow *", "osd", "allow *"}
```

Comment:
The open MGR permissions are for the upcoming Ceph MGR rook module, right?

Reply:
yes, it was also needed for the mimic dashboard iirc